### ISSUE TITLE
high learning rate in later stages

### ISSUE THREAD

**Issue by coldgemini** [shihenw/convolutional-pose-machines-release#66]
Hi,
why did you use much higher (4X) learning rate from 2 stages and leave only the first stage a small learning rate? If it's kind of a fine tune strategy, why did you not just freeze the first stage? Is it due to some empirical results?

Thank you a lot.

**Comment by 1Rookiechang** [comment]
have you download the pretrained model？can you share it with me.
because the download site cannot open.thank you very much!
827927604@qq.com
